### PR TITLE
removed technet link added docs.microsoft link

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #2943, so i removed Technet link and added docs.microsoft.com  link

**https://docs.microsoft.com/previous-versions/windows/it-pro/windows-8.1-and-8/jj131725(v=ws.11)?redirectedfrom=MSDN**

### CODE REVIEW

**Review by get-itips — CHANGES_REQUESTED** [review]
Looks like there is a more updated article

https://docs.microsoft.com/en-us/windows/security/information-protection/tpm/trusted-platform-module-overview

File: docset/winserver2022-ps/trustedplatformmodule/Disable-TpmAutoProvisioning.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Provisioning is the process of preparing a TPM to be used.
 You can disable provisioning completely or only for the next restart.
 You can use the Enable-TpmAutoProvisioning cmdlet to enable auto-provisioning.
 
-For more information on TPM, see the [Trusted Platform Module Technology Overview](https://technet.microsoft.com/en-us/library/jj131725.aspx) in the Technet library.
+For more information on TPM, see the [Trusted Platform Module Technology Overview](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-8.1-and-8/jj131725(v=ws.11)?redirectedfrom=MSDN) in the Technet library.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/trustedplatformmodule/Disable-TpmAutoProvisioning.md:28]:
```suggestion
For more information on TPM, see [Trusted Platform Module Technology Overview](/windows/security/information-protection/tpm/trusted-platform-module-overview).
```

**Comment by get-itips** [comment]
Can you check the other articles for different OS versions?

**Review by tseward — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**